src/views/ConfigurationJoystickView.vue

```diff
@@ -759,6 +759,14 @@ const filteredAndSortedJoystickActions = computed((): JoystickAction[] => {
     .sort((a: JoystickAction, b: JoystickAction) => a.name.localeCompare(b.name))
 })
 
+const filteredAndSortedAxisActions = computed((): JoystickAction[] => {
+  return controllerStore.availableAxesActions.filter((action: JoystickAction) => {
+    const dataLakeVariableInfo = getDataLakeVariableInfo(action.id)
+    if (!dataLakeVariableInfo) return true
+    return dataLakeVariableInfo.allowUserToChangeValue && dataLakeVariableInfo.type !== 'string'
```

Comment:
This implies boolean data-lake variables can be used, which seems reasonable (e.g. to convert an axis to a button if you want to), but is not handled correctly because the number just gets assigned regardless of the variable's declared type.

Reply:
Will filter to only show `number` ones for now.

Reply:
Fair enough. It's still possible to create some messes by changing the type of a variable after assignment somewhere, but that seems fair to consider as just excessive user error for now 👍